### ISSUE TITLE
1.5 hours course

### ISSUE THREAD

**Issue by anish-kum** [anish-kum/PPP-Spark-2025-2026#6]
PPP 90 minutes

**Comment by anish-kum** [comment]
<img width="66" height="32" alt="Image" src="https://github.com/user-attachments/assets/331eab92-d467-4596-8c9a-855e75b36786" />

Start Time: 10:25 PM

**Comment by anish-kum** [comment]
<img width="327" height="502" alt="Image" src="https://github.com/user-attachments/assets/90ba2729-3061-4814-8fe7-5aa0c32634e4" />

<img width="332" height="110" alt="Image" src="https://github.com/user-attachments/assets/ce0d5713-2737-4316-a54f-9e74389e9119" />

What I was learning:

- Cross-Entropy Loss: Measures the difference between true and predicted probability distributions.

- Classification Check: Evaluates whether a point is correctly classified.

- Softmax Function: Converts raw output scores into probabilities.

- Gradient Descent: Optimizes model parameters by minimizing the loss function through iterative updates based on computed gradients.

- Iterative Learning: The perceptron trick may need to be applied multiple times until the model correctly classifies the point.

I definitely need to go through these again over the next week, as there's a lot I need more practice with, since it's easy to forget this material without regular reviewing.

**Comment by anish-kum** [comment]
<img width="64" height="29" alt="Image" src="https://github.com/user-attachments/assets/f21c23e3-3486-4299-8481-9d5623266e86" />

End Time: 12 AM

**Comment by drewmca-pkwy** [comment]
@anish-kum Good summary, Anish. I also appreciate the note at the end. It gives me a sense of how you're interacting with this content. As you've settled into a rhythm with this work, I have one more requirement for these cards: at least one question the information did NOT cover that you'd like an answer to. Recording the confusions that come up allow me to track how you're connecting this information to contexts outside the course.
